File: Competitive Coding - 1/3. Problems on Mathematics/Q4.py
import sys

a = int(sys.argv[1])
b = int(sys.argv[2])

def getProductSign(a, b):
	# write code from here 
	if a>0 and b>0: return 'positive'
	if a<=0<=b:
		return 'zero'
	negatives = a-b+1
	if negatives%2==0:
		return'positive'
	return 'negative'
	# ends here

print(getProductSign(a, b))

File: Competitive Coding - 1/1. Problems on Array/Q4.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
int maxSumContiguousSubArray(int arr[], int arrLen) {
    // write code from here
    int max=0;
    int maxend=0;
    for(int i=0;i<arrLen;i++){
        maxend=maxend+arr[i];
        if(maxend<0){
            maxend=0;
        }
        if(max<maxend){
            max=maxend;
        }
    }
    return max;
    //ends here
}
int readIntArray(char *argsArray, int arr[]) {
    int col = 0;
    char *token = strtok(argsArray, ",");
    while (token != NULL) {
        arr[col] = atoi(token);
        token = strtok(NULL, ",");
        col++;
    }
    return col;
}
int main(int argc, char *argv[]) {
